cmd/create/execution.go

```diff
@@ -78,6 +78,12 @@ Also an execution can be relaunched by passing in current execution id.
 
  flytectl create execution --relaunch ffb31066a0f8b4d52b77 -p flytectldemo -d development
 
+An execution can be recovered, that is recreated from the last known failure point for a previously-run workflow execution.
```

Comment:
i think this is very important to explain the difference from relaunch. Having a single line is probably not enough. Can we intersphinx link?

Reply:
This is parsed as rst text so it should be possible i think.

Reply:
done